openassessment/xblock/defaults.py

```diff
@@ -63,13 +63,13 @@
 # The rubric's feedback prompt is a set of instructions letting the student
 # know they can provide additional free form feedback in their assessment.
 DEFAULT_RUBRIC_FEEDBACK_PROMPT = """
-(Optional) What aspects of this response stood out to you? What did it do well? How could it improve?
+(Optional) What aspects of this response stood out to you? What did it do well? How could it be improved?
```

Comment:
I'm looking into the indentation now, that struck me as odd also.

As far as the old/new text, changing the defaults will only affect *new* problems, as old ones have already been saved with the previous default value. Is this a huge problem?

<img width="856" alt="screen shot 2016-01-20 at 3 43 29 pm" src="https://cloud.githubusercontent.com/assets/7373924/12462518/c3b59784-bf8c-11e5-8b79-e29ff4bc82fd.png">